examples/tutorials/03_vision/03_annotate.py

```diff
@@ -65,8 +65,7 @@ def apply(self, image, scale):
         text.render(d, bounds)
 
 
-async def run(sdk_conn):
-    robot = await sdk_conn.wait_for_robot()
+async def cozmo_program(robot: cozmo.robot.Robot):
```

Comment:
so this will be the first example that uses async... should we add some additional comments to it?   or we can convert it to sync